### PR TITLE
GitOps- Renamed Application Stages in nav to Environments

### PR DESCRIPTION
**Pull request by reginapizza**
Environments
Environments- ApplicationEnvironments

This change is only to rename the "Application Stages" section of the nav bar to be "Environments". GitOpsDetailsPage is also now shown as "Application Environments" instead of "Application Details".

@serenamarie125

### CODE REVIEW

**Review by serenamarie125 — APPROVED** [review]
Looks great @reginapizza ! FYI @siamaksade

**Comment by divyanshiGupta** [comment]
@reginapizza change `Details` to `Environments` here https://github.com/openshift/console/blob/548aa3395d7073ac45803bc76b9bf86683642a87/frontend/packages/dev-console/src/components/gitops/GitOpsDetailsPage.tsx#L78

**Comment by andrewballantyne** [comment]
> @reginapizza: The following tests **failed**, say `/retest` to rerun all failed tests:
> 
> ci/prow/frontend	[102be80](https://github.com/openshift/console/commit/102be8094820b948b127d9897b49ffc2e7bd8109)	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_console/7272/pull-ci-openshift-console-master-frontend/1333426952081510400)	`/test frontend`

You'll need to run the I18n command in the package.json in order to get past the error in `frontend`.

File: frontend/packages/dev-console/locales/en/devconsole.json [hunk 1]
```diff
@@ -448,4 +448,4 @@
   "Operator Backed": "Operator Backed",
   "Browse the catalog to discover and deploy operator managed services": "Browse the catalog to discover and deploy operator managed services",
   "Browse the catalog to discover and install Helm Charts": "Browse the catalog to discover and install Helm Charts"
-}
\ No newline at end of file
+}
```
Comment (divyanshiGupta) [inline frontend/packages/dev-console/locales/en/devconsole.json:451]: @reginapizza after running `yarn i18n` do not modify this file to add a new line.

Comment (reginapizza) [reply frontend/packages/dev-console/locales/en/devconsole.json]: sorry, must have added that line automatically. I've removed it now.

File: frontend/packages/dev-console/locales/en/devconsole.json [hunk 2]
```diff
@@ -63,15 +63,15 @@
   " by ": " by ",
   "Commit details not available": "Commit details not available",
   "Cluster URL not available": "Cluster URL not available",
-  "Application Stages": "Application Stages",
-  "Application Details": "Application Details",
+  "Environments": "Environments",
+  "Application Environments": "Application Environments",
```
Comment (christianvogt) [inline frontend/packages/dev-console/locales/en/devconsole.json:67]: Looks like this string isn't being used. The frontend tests are failing.
Run `i18n` again.

**Comment by divyanshiGupta** [comment]
@reginapizza also please squash your commits into one.

File: frontend/packages/dev-console/src/components/gitops/details/GitOpsDetailsPageHeading.tsx [hunk 3]
```diff
@@ -21,11 +21,11 @@ const GitOpsDetailsPageHeading: React.FC<GitOpsDetailsPageHeadingProps> = ({
...
   const breadcrumbs = [
     {
-      name: t('devconsole~Application Stages'),
-      path: '/applicationstages',
+      name: t('devconsole~Environments'),
+      path: '/environments',
     },
     {
-      name: t('devconsole~Application Details'),
+      name: t('devconsole~Environments'),
```
Comment (keithchong) [inline frontend/packages/dev-console/src/components/gitops/details/GitOpsDetailsPageHeading.tsx:28]: Should this be devconsole~Application Environments?  If so, this _should_ fix the test.

Comment (divyanshiGupta) [reply frontend/packages/dev-console/src/components/gitops/details/GitOpsDetailsPageHeading.tsx:28]: Yes this should be `devconsole~Application Environments`. @reginapizza please fix this.

**Pull request merged**